operator/pkg/apis/core/customize/v1beta1/controllerresource_types.go

```diff
@@ -56,6 +56,12 @@ type ContainerResourceSpec struct {
 	// Resources specifies the resource customization of this container.
 	// Required
 	Resources ResourceRequirements `json:"resources"`
+
+	// RateLimit describes the token bucket rate limit to the kubernetes client.
+	// Please note this rate limit is shared among all the Config Connector resources' requests.
+	// If not specified, the default will be Token Bucket with qps 20, burst 30.
+	// +optional
+	RateLimit *RateLimit `json:"rateLimit,omitempty"`
```

Comment:
The Config Connector rate limit is actually in the container `arg` (See below) rather than some `rateLimit` field.  But I want to be careful on user configurable fields, specifically I don't want to expose the access of other container args in the specification. So I defined a new field here that will append/override the `args`.